assets/less/site/rdm/collections/grid.overrides

```diff
@@ -0,0 +1,6 @@
+.ui.grid.subgrid {
+  .column {
+    padding-left: 0;
```

Comment:
Do not reset padding on every possible grid column, use p-0 class where needed instead of this